### PR TITLE
[SPARK-15305][ML][DOC]:spark.ml document Bisectiong k-means has the incorrect format

### PR DESCRIPTION
**Pull request by wangmiao1981**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)
The generated document has the incorrect format for biseckmeans.

bug
## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

Fix the formatting.
fix

### CODE REVIEW

**Comment by wangmiao1981** [comment]
cc @zhengruifeng @yanboliang

**Comment by zhengruifeng** [comment]
LGTM

File: docs/ml-clustering.md [hunk 1]
```diff
@@ -116,6 +116,8 @@ Refer to the [Python API docs](api/python/pyspark.ml.html#pyspark.ml.clustering.
 {% include_example python/ml/lda_example.py %}
 </div>
 </div>
+
+
```
Comment (MLnick) [inline docs/ml-clustering.md]: Is 1 line enough? Does it need to be 2?

Comment (wangmiao1981) [inline docs/ml-clustering.md]: I followed the style of existing items like LDA, Kmeans etc. Let me try 1line.

**Comment by wangmiao1981** [comment]
@MLnick I removed all other blank lines in the ml-clustering.md. One blank line works.

**Comment by MLnick** [comment]
Thanks, merged to master/branch-2.0.

**Pull request closed without merging**